### PR TITLE
feat(extras.lang): add jsonnet support

### PR DESCRIPTION
**Pull request by cbosvik**

### CODE REVIEW

File: lua/lazyvim/plugins/extras/lang/jsonnet.lua [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    end,
+  },
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed, {
+        "jsonnet-language-server",
+      })
+    end,
+  },
```
Comment (dpetka2001) [inline lua/lazyvim/plugins/extras/lang/jsonnet.lua:27]:
```suggestion
```
This is not needed, as LazyVim automatically installs the servers that are defined in `nvim-lspconfig.opts.servers`. 
For more info, how this is set up, see [here](https://github.com/LazyVim/LazyVim/blob/97480dc5d2dbb717b45a351e0b04835f138a9094/lua/lazyvim/plugins/lsp/init.lua#L201-L216).

**Review by dpetka2001 — CHANGES_REQUESTED** [review]

File: lua/lazyvim/plugins/extras/lang/jsonnet.lua (hunk 1 above)
Comment (cbosvik) [reply lua/lazyvim/plugins/extras/lang/jsonnet.lua]: Oh, I wasn't aware, Thanks

**Review by cjalvis — APPROVED** [review]

**Comment by dpetka2001** [comment]
@cjalvis kindly stop approving. You also double approve, so the needless notifications come double in the inbox after I clear them. Please just thumb up the PR.

**Comment by cjalvis** [comment]
Yeah my bad.

**Pull request closed without merging**